modules/node/persistence/cms_node_storage_sql.e

```diff
@@ -335,6 +343,16 @@ feature {NONE} -- Implementation
 				if attached sql_read_date_time (11) as l_modif_date then
 					Result.set_modification_date (l_modif_date)
 				end
+				if attached sql_read_integer_32 (12) as l_status then
+					inspect l_status
+					when {CMS_NODE_CONSTANTS}.not_published then
+						Result.mark_not_published
+					when {CMS_NODE_CONSTANTS}.published then
+						Result.mark_published
+					when {CMS_NODE_CONSTANTS}.trash then
+						Result.mark_trash
```

Comment:
add a default "else" part to avoid crash if l_status has wrong value.
and also, if a module introduce a new status, this code will be broken.
Maybe we should not add too much knowledge here, and simply do a set_status (l_status)